Datadog/Example/AppConfig.swift

```diff
@@ -12,13 +12,16 @@ protocol AppConfig {
     var serviceName: String { get }
     /// SDK configuration
     var datadogConfiguration: Datadog.Configuration { get }
+    /// Endpoint for arbitrary network requests
+    var sourceEndpoint: URL { get }
 }
 
 struct ExampleAppConfig: AppConfig {
     /// Service name used for logs and traces.
     let serviceName = "ios-sdk-example-app"
     /// Configuration for uploading logs to Datadog servers
     let datadogConfiguration: Datadog.Configuration
+    let sourceEndpoint = URL(string: "https://app.datadoghq.com/")!
```

Comment:
As our backend monitoring intensively watches the DD entry point: https://app.datadoghq.com/, maybe we can use more "technical" URL for this? How about https://status.datadoghq.com?

Reply:
set to Datadog logo image URL ✅